gsk4/sys/src/lib.rs

```diff
@@ -109,7 +109,7 @@ pub const GSK_TRANSFORM_CATEGORY_IDENTITY: GskTransformCategory = 6;
 
 // Callbacks
 pub type GskParseErrorFunc =
-    Option<unsafe extern "C" fn(gconstpointer, *const glib::GError, gpointer)>;
+    Option<unsafe extern "C" fn(*const gtk::GtkCssSection, *const glib::GError, gpointer)>;
```

Comment:
Does that compile? :)

Reply:
I don't know for now, waiting on the gtk4 docker image to get updated